pkg/handler/eval.go

```diff
@@ -220,14 +223,14 @@ var evalSegment = func(
 				Msg:       err.Error(),
 				SegmentID: int64(segment.ID),
 			}
-			return nil, log
+			return nil, log, true
```

Comment:
What kinds of errors do we expect from `conditions.Evaluate(expr, m)`?  Trying to understand why we should eval the next segment in this context